src/main/cli/cli.c

```diff
@@ -6241,6 +6252,9 @@ static void cliHelp(char *cmdline);
 // should be sorted a..z for bsearch()
 const clicmd_t cmdTable[] = {
     CLI_COMMAND_DEF("adjrange", "configure adjustment ranges", "<index> <unused> <range channel> <start> <end> <function> <select channel> [<center> <scale>]", cliAdjustmentRange),
+#ifdef USE_TUNING_SLIDERS
+    CLI_COMMAND_DEF("apply_tuning_sliders", "applies tuning sliders calculation based on slider positions", NULL, cliTuningSliders),
```

Comment:
I'm not sure there's value in having the `apply_tuning_sliders` CLI command. When would it be used? Would we be expecting the user to `set` the slider values, be sure to `apply_tuning_sliders`, then `save`? Seems like we should apply the slider calculations automatically in `validateAndFixConfig()`.

Reply:
@etracer65: This will be problematic as this will result in other tuning values being overwritten on config restore, making it impossible to use any of the independent settings to fine-tune the output of the sliders.